trojsten/notifications/management/commands/sendnotifications.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
+from django.utils.translation import ugettext_lazy as _
+
+from trojsten.notifications.models import Notification
+from trojsten.people.models import User
+
+
+class Command(BaseCommand):
```

Comment:
Na co ma tento command presne sluzit? Potrebujeme ho?

Reply:
odosielanie mailov, ideálne by mal bežať v nejakom crone

Reply:
@black3r chceme toto konfigurovat? Alebo spravime proste ten docker s celery? Alebo je ok poslat mrt (~100?) mailov pocas requestu zatial kym nespravime docker s celery.

Adam, jedna z alternativ je zatial mat tie notifikacie bez mailov a potom posielat maily ked to bude bezat z celery. Ak sa nedohodneme ze sa nam to chce na servri konfigurovat, tak je toto zbytocne. Pockal by som na Romana ze co povie.

Reply:
Dá sa to potom prerobiť, aktuálne ak to nebudeš spúšťať, tak maily nebudú :)

Reply:
Ok.